laghos_rom.hpp

```diff
@@ -78,6 +78,11 @@ struct ROM_Options
     int dimFv = -1;
     int dimFe = -1;
 
+    // Incremental SVD options
+    double model_linearity_tol = 1.e-7;
```

Comment:
I think these are the historical names of these variables. Maybe it is a time to update them to something more descriptive. First, `model` is not helpful. Second, if they are only used for incremental SVD, how about something like `model_*` --> `incSVD_*`?